packages/compiler/src/index.ts

```diff
@@ -96,6 +96,7 @@ export function compileVineTypeScriptFile(
   code: string,
   fileId: string,
   vineCompileCtx: VineCompileCtx,
+  ssr: boolean,
```

Comment:
Maybe set default value for `ssr = false` ? This allows us to omit passing false from those previous non-SSR test code

Reply:
> Maybe set default value for `ssr = false` ? This allows us to omit passing false from those previous non-SSR test code

Agree! Let me do it.